README.md

```diff
@@ -20,81 +20,47 @@ $ aws-okta add
-
-```bash
-$ aws-okta help exec
-exec will run the command specified with aws credentials set in the environment
-
-Usage:
-  aws-okta exec <profile> -- <command>
-
-Flags:
-  -a, --assume-role-ttl duration   Expiration time for assumed role (default 1h0m0s)
-  -h, --help                       help for exec
-  -t, --session-ttl duration       Expiration time for okta role session (default 1h0m0s)
-
-Global Flags:
-  -b, --backend string   Secret backend to use [kwallet secret-service file] (default "file")
-  -d, --debug            Enable debug logging
-```
-
-### Exec for EKS and Kubernetes
-
-`aws-okta` can also be used to authenticate `kubectl` to your AWS EKS cluster. Assuming you have [installed `kubectl`](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html), [setup your kubeconfig](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html) and [installed `aws-iam-authenticator`](https://docs.aws.amazon.com/eks/latest/userguide/configure-kubectl.html), you can now access your EKS cluster with `kubectl`. Note that on a new cluster, your Okta CLI user needs to be using the same assumed role as the one who created the cluster. Otherwise, your cluster needs to have been configured to allow your assumed role.
-
-```bash
-$ aws-okta exec <profile> -- kubectl version --short
-```
-
-Likewise, most Kubernetes projects should work, like Helm and Ark.
-
-```bash
-$ aws-okta exec <profile> -- helm version --short
-```
+### Configuring your AWS config
 
-### Configuring your aws config
-
-`aws-okta` assumes that your base role is one that has been configured for Okta's SAML integration by your Okta admin. Okta provides a guide for setting up that integration [here](https://support.okta.com/help/servlet/fileField?retURL=%2Fhelp%2Farticles%2FKnowledge_Article%2FAmazon-Web-Services-and-Okta-Integration-Guide&entityId=ka0F0000000MeyyIAC&field=File_Attachment__Body__s).  During that configuration, your admin should be able to grab the AWS App Embed URL from the General tab of the AWS application in your Okta org.  You will need to set that value in your `~/.aws/config` file, for example:
+The most seamless way to use `aws-okta` is via AWS SDK config credential
+process. To do so you will need to setup your AWS config as `aws-okta` assumes
+that your base role is one that has been configured for Okta's SAML integration
+by your Okta admin. Okta provides a guide for setting up that integration
+[here](https://support.okta.com/help/servlet/fileField?retURL=%2Fhelp%2Farticles%2FKnowledge_Article%2FAmazon-Web-Services-and-Okta-Integration-Guide&entityId=ka0F0000000MeyyIAC&field=File_Attachment__Body__s).
+During that configuration, your admin should be able to grab the AWS App Embed
+URL from the General tab of the AWS application in your Okta org.  You will need
+to set that value in your `~/.aws/config` file, for example:
 
 ```ini
-[okta]
+[profile okta]
```

Comment:
`[okta]` is a special TOML config block for okta specific defaults (like the saml URL). It's probably a bit confusing if you combine the `[okta]` block with the `[profile]` block and call the profile `okta` as well.

I'd at least rename this profile to something else.

Reply:
I was wondering if it was something like that. I haven't used it before 😂  I think that is where some of my confusion came from.